tools/serverpod_cli/test/test_util/mock_std.dart

```diff
@@ -0,0 +1,281 @@
+import 'dart:async';
+import 'dart:convert';
+import 'dart:io';
+
+class MockStdout implements Stdout {
+  final _buffer = StringBuffer();
```

Comment:
Will come in handy. But perhaps something to share with cli_tools.. or cli_tools_test. (For later)